scenarios/tracking/02_mot_challenge.ipynb

```diff
@@ -0,0 +1,368 @@
+{
```

Comment:
<p>The image does not show. I see you added a file to <a href="https://github.com/microsoft/computervision-recipes/pull/569/files#diff-63399ed988edbd9c626e61e0e7142766" target="_blank">scenarios/media/mot_results.PNG. </a>Please instead move under scenarios/tracking/media and if the file is larger then convert to jpg.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>